packages/users/src/users.spec.ts

```diff
@@ -452,5 +452,30 @@ describe('Users...', () => {
         }, 350)
       }, 500)
     }).timeout(5000)
+
+  })
+  describe('usage', () => {
+    const ctx = new Context(addrApiurl)
+    let dev: SignupResponse.AsObject
+    before(async function () {
+      this.timeout(10000)
+      const { user } = await signUp(ctx, addrGatewayUrl, sessionSecret)
+      if (user) dev = user
+    })
+    it('it should return the correct usage per user', async () => {
+      const tmp = new Context(addrApiurl).withSession(dev.session)
+      const { keyInfo } = await createKey(tmp, 'KEY_TYPE_ACCOUNT')
+      await ctx.withAPIKey(keyInfo?.key).withKeyInfo(keyInfo)
+      const user = new Users(ctx)
+      try {
```

Comment:
This is so cool!

Reply:
ah @sanderpick to answer your question about tests, not very much. i only pull of testing the successful error here. not sure how to do the full billing enabled setup.